documentation/TESTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
```

Comment:
Hi @patmejia thanks for adding these instructions! Can I suggest this file may be better off in the project root directory rather than in a subfolder? As well, maybe this can form the basis of a DEVELOPMENT.md so we can start consolidating developer instructions from the readme and from testing into one document?